x/auth/client/rest/query.go

```diff
@@ -92,11 +86,6 @@ func QueryTxsRequestHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
-		//append tx height events
-		if txHeightEvents != nil && len(txHeightEvents) > 0 {
-			events = append(events, txHeightEvents...)
-		}
-
 		searchResult, err := utils.QueryTxsByEvents(cliCtx, events, page, limit)
```

Comment:
where is code for appending events? It loooks to me that there is only validation.

Reply:
https://github.com/terra-project/core/blob/8a3751eb7fd4fde724556d77b7280d4f7b0e974d/x/auth/client/rest/query.go#L73

Now we don’t need to support our custom height options. Cosmos will do that with minheight and maxheight flag

Reply:
https://github.com/terra-project/core/blob/8a3751eb7fd4fde724556d77b7280d4f7b0e974d/x/auth/client/rest/query.go#L73

Within this function, minheight and maxheight and height will be interpreted to tendemint events.

Reply:
https://github.com/cosmos/cosmos-sdk/blob/2727c14183d05f5c31ca11f2dcd52b0aa683fadf/types/rest/rest.go#L220